scripts/populate-popular-and-new-and-noteworthy-projects.py

```diff
@@ -0,0 +1,140 @@
+"""
+This will update node links on POPULAR_LINKS_NODE and NEW_AND_NOTEWORTHY_LINKS_NODE.
+"""
+import sys
+import logging
+import datetime
+import dateutil
+import operator
+import requests
+from modularodm import Q
+from website.app import init_app
+from website import models
+from framework.auth.core import Auth
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+from website.settings.defaults import POPULAR_LINKS_NODE, NEW_AND_NOTEWORTHY_LINKS_NODE
```

Comment:
Ooooh! This is interesting! We may indeed have some automatic layering of `website.settings.local` values into `website.settings.default`, because I believe I'm seeing that if I set `POPULAR_LINKS_NODE` in `website.settings.local`, that is what gets hit from the client.